core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -236,11 +236,6 @@ Optional<TableExecuteHandle> getTableHandleForExecute(
      */
     void setViewColumnComment(Session session, QualifiedObjectName viewName, String columnName, Optional<String> comment);
 
-    /**
```

Comment:
Can we reword the commit as `Rearrage  setMaterializedViewColumnComment method order`. Can we extract the test changes as a separate commit ?

Reply:
refactored